net-im/skypeforlinux/skypeforlinux-9999.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils rpm
 
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
-SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
+SRC_URI="https://repo.skype.com/latest/skypeforlinux-64-alpha.rpm"
```

Comment:
https://repo.skype.com/rpm/stable/

`cmp` confirms that it's the same RPM as gets fetched by latest.